jack_build.py

```diff
@@ -141,7 +141,8 @@
 const char** jack_port_get_all_connections(const jack_client_t* client, const jack_port_t* port);
 /* deprecated: jack_port_tie */
 /* deprecated: jack_port_untie */
-int jack_port_set_name(jack_port_t* port, const char* port_name);
```

Comment:
You could leave the declaration in there (but also keep the comment marking it as deprecated).

This way, users who would like to use it (for whatever reason) can still call it from their code using the `_lib` object.

And I think the declaration doesn't hurt, even if in the future this function will completely be removed from the JACK library.

Reply:
@lack the deprecated info for the old function would still be desirable. This is still missing.

`int jack_port_set_name(jack_port_t* port, const char* port_name);  /* deprecated */`